ci/pipelines/cpi-release-pipeline.yml

```diff
@@ -138,21 +138,22 @@ jobs:
 
       - <<: *terraform-setup
       - <<: *terraform-apply
-      - <<: *terraform-output
       - <<: *prepare-test-config
       - do:
         - <<: *setup-director
+        - <<: *run-bats
         ensure:
           do:
             - <<: *teardown-director
             - <<: *terraform-destroy
 
   - name: build-final
+    serial_groups: [oci-resource-consumer]
     serial: true
     plan:
       - aggregate:
          - {trigger: false, get: final-version-semver,   params: {bump: major}}
-         - {trigger: true, passed: [run-ocitests, create-director], get: cpi-release-src, resource: bosh-cpi-release-src}
+         - {trigger: true, passed: [build-candidate], get: cpi-release-src, resource: bosh-cpi-release-src}
```

Comment:
This seems incorrect to me.  Do we really want to produce a final build without ensuring that ocitests passed correctly? I can understand not gating the final build on run-bats until that test suite is clean but not sure why are we skipping the ocitests that have been running cleanly for a while now.

Reply:
Never mind, I see that run-ocitests is executed prior  to build candidate.